src/components/sidebar/SidebarHeader/index.tsx

```diff
@@ -29,18 +28,6 @@ import Track from '@/components/common/Track'
 import { OVERVIEW_EVENTS } from '@/services/analytics/events/overview'
 import { SvgIcon } from '@mui/material'
 
-const HeaderIconButton = ({
```

Comment:
Why did we need to remove the abstraction?

Reply:
After removing the redundant `a` element, the `IconButton` components were different for the 2 places. We could pass those props to the abstraction but imo its not worth the added complexity.